### PR TITLE
DOC Update SilverStripe to Silverstripe CMS

### PR DESCRIPTION
**Pull request by Michael-HEIW**
- Remaining Developer Guides and Upgrading
- SilverStripe in a namespace or api has not been change
- To keep PRs easier no formatting was changed

### CODE REVIEW

**Review by NightJar — APPROVED** [review]
I'll approve this, because it's great work, great effort, thanks!
However there are a couple of questions, mostly not concerning the changes themselves and certainly things that can be addressed after this is merged... however I'd like to take the opportunity to point out a couple of phrasing questions and potential areas in need of adjustment to some other folk with domain over the docs here.

If there's no clear direction in a day or two and I've not merged please remind me :)

There are a couple of small non-critical adjustments you could do in the meantime though:
- consider squashing the branch update commit
- change the reference that was "SilverStripe Platform" to "Silverstripe Cloud" (comment on it to find it easily)

Other than that this is amazing, thanks!

File: docs/en/02_Developer_Guides/00_Model/12_Indexes.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ The addition of an indexes should be carefully evaluated as they can also increa
...
 E.g. In most cases an index on `boolean` status flag, or `ENUM` state will not increase query performance.
 
-It's important to find the right balance to achieve fast queries using the optimal set of indexes; For SilverStripe 
+It's important to find the right balance to achieve fast queries using the optimal set of indexes; For Silverstripe CMS 
 applications it's a good practice to: 
 - add indexes on columns which are frequently used in `filter`, `where` or `orderBy` statements
 - for these, only include indexes for columns which are the most restrictive (return the least number of rows)
 
-The SilverStripe framework already places certain indexes for you by default:
+The Silverstripe CMS framework already places certain indexes for you by default:
```
[the remark this answers is not in the document]
Comment (brynwhyman) [reply docs/en/02_Developer_Guides/00_Model/12_Indexes.md:20]: According to the brand guidelines, 'Silverstripe CMS' is a replacement for all of the following terms:
- SilverStripe
- SilverStripe CMS
- Silverstripe Framework

My opinion is a lowercase 'f' in framework is still fine to use.

File: docs/en/02_Developer_Guides/02_Controllers/02_Routing.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ class BreadAPIController extends Controller
 ```
 
 <div class="alert" markdown="1">
-In SilverStripe Framework versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
+In Silverstripe CMS Framework versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/02_Controllers/02_Routing.md:285]: Given Bryn's response on a previous comment, it is probably best to remove the term `Framework` from this sentence.

File: docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Built-in Middleware
-summary: Middleware components that come with SilverStripe Framework
+summary: Middleware components that come with Silverstripe CMS Framework
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md:3]: remove `Framework`

File: docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 ---
 
 # Built-in Middleware
 
-SilverStripe Framework has a number of Middleware components.
+Silverstripe CMS Framework has a number of Middleware components.
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md:8]: remove `Framework`

File: docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
+Silverstripe CMS is an application framework which can be used to process
 and store data. Any data can be sensitive, particularly if it is
 considered personal data. Many regulatory frameworks such as the
 [EU General Data Protection Regulation (GDPR)](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation)
 can be interpreted to regard basic data points such as email and IP addresses
 as personally identifiable data.
 
 This document is aiding implementors and auditors in determining
-the impact of using the SilverStripe Framework and CMS
+the impact of using the Silverstripe CMS Framework and CMS
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md:17]: please change to `[...] using the Silverstripe CMS to build online services. [...]` (remove `Framework and CMS`)

File: docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md [hunk 6]
```diff
@@ -1,32 +1,32 @@
...
 to build online services. Since every website and app built with
-SilverStripe will be different, it can only provide starting points.
+Silverstripe CMS will be different, it can only provide starting points.
 
 ## Storage
 
-The SilverStripe CMS does not provide any built-in mechanisms for users to submit personal data,
+The Silverstripe CMS does not provide any built-in mechanisms for users to submit personal data,
```
[the remark this answers is not in the document]
Comment (NightJar) [reply docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md:23]: As responded elsewhere, brand guidelines encompass "Silverstripe CMS" (as a module) in the terms that are to be replaced with "Silverstripe CMS" (as a system).

File: docs/en/02_Developer_Guides/14_Files/05_File_Migration.md [hunk 7]
```diff
@@ -90,7 +90,7 @@ It attempts to continue running to "complete" status even if it encounters error
 to ensure if everything went smoothly. Note that it's currently not possible to run specific subtasks via a queuedjob.
 
 While you can run the job directly through the CMS, it'll usually be more constrained by PHP `max_execution_time` settings.
-Many platforms such as the New Zealand Government Common Web Platform or SilverStripe Platform
+Many platforms such as the New Zealand Government Common Web Platform or Silverstripe CMS Platform
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/14_Files/05_File_Migration.md:93]: Please change `Silverstripe CMS Platform` to `Silverstripe Cloud` :)

File: docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/index.md [hunk 8]
```diff
@@ -5,11 +5,11 @@ introduction: The Admin interface can be extended to provide additional function
 iconBrand: react
 ---
 
-The Admin interface is bundled within the SilverStripe Framework but is most commonly used in conjunction with the `cms`
+The Admin interface is bundled within the Silverstripe CMS Framework but is most commonly used in conjunction with the `cms`
```
Comment (NightJar) [inline docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/index.md:8]: Please remove `Framework`

**Review by NightJar — CHANGES_REQUESTED** [review]
Sorry to do this, but given [Bryn's recent feedback](https://github.com/silverstripe/silverstripe-framework/pull/10003#discussion_r673605976) around more specific usage of the term "Silverstripe CMS" and the terms it replaces, I feel there are a few extra edits to be made.

There are a couple of outstanding edits to be done also, I'll comment on the ones I think should be done.

So very close! After addressing all the feedback I'll be most happy to finally merge :)

File: docs/en/02_Developer_Guides/14_Files/05_File_Migration.md [hunk 9]
```diff
@@ -210,82 +210,82 @@ Use the following estimates to decide how you will run your file migration:
 | < 10000 | Command Line | 10000 seconds | 950 MB |
 | 10000+ | Command Line or contact support | n/a | n/a |
 
-Your exact experience will vary based on your host server, the size of your files and other conditions. If your site is hosted on a managed environement (e.g.: [Common Web Platform](https://www.cwp.govt.nz/service-desk) or [SilverStripe Platform](https://docs.platform.silverstripe.com/support/)), you may not have access to the command line to manually run the migration task. Contact your hosting provider's helpdesk if that's your case.
+Your exact experience will vary based on your host server, the size of your files and other conditions. If your site is hosted on a managed environement (e.g.: [Common Web Platform](https://www.cwp.govt.nz/service-desk) or [Silverstripe CMS Platform](https://docs.platform.silverstripe.com/support/)), you may not have access to the command line to manually run the migration task. Contact your hosting provider's helpdesk if that's your case.
```
[the remark this answers is not in the document]
Comment (NightJar) [reply docs/en/02_Developer_Guides/14_Files/05_File_Migration.md:213]: Please alter the "Silverstripe Platform" name to "Silverstripe Cloud"

File: docs/en/00_Getting_Started/02_Composer.md [hunk 10]
```diff
@@ -54,9 +54,9 @@ see [Using development versions](#using-development-versions).
 
 ## Adding modules to your project
 
-Composer isn't only used to download Silverstripe CMS, it is also used to manage all Silverstripe modules. You can find
-thousands of modules on [https://addons.silverstripe.org](https://addons.silverstripe.org). Installing a module can be
-done with the following command:
+Composer isn't only used to download Silverstripe CMS, it is also used to manage all SilverStripe modules.
```
[the remark this answers is not in the document]
Comment (NightJar) [reply docs/en/00_Getting_Started/02_Composer.md:57]: Please also alter the second occurrence of `SilverStripe` to `Silverstripe CMS` in this sentence.

**Comment by NightJar** [comment]
Doing another rebase will hopefully resolve the build error too :)

(presuming it comes from the branch point commit, as you've changed no code to cause a build failure).

File: docs/en/00_Getting_Started/02_Composer.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (Michael-HEIW) [reply docs/en/00_Getting_Started/02_Composer.md:57]: Updated

File: docs/en/02_Developer_Guides/14_Files/05_File_Migration.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/14_Files/05_File_Migration.md:213]: Updated to "Silverstripe Cloud"

File: docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md (hunk 4 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md:8]: Framework removed

File: docs/en/00_Getting_Started/02_Composer.md [hunk 11]
```diff
@@ -323,8 +323,8 @@ on [packagist.org](http://packagist.org).
 ### How should I name my module?
 
 Follow the packagist.org advice on choosing a [unique name and vendor prefix](https://packagist.org/about). Please don't
-use the `silverstripe/<modulename>` vendor prefix, since that's reserved for modules produced by Silverstripe Ltd. In
-order to declare that your module is in fact a Silverstripe module, use the "silverstripe" tag in the composer.json
+use the `silverstripe/<modulename>` vendor prefix, since that's reserved for modules produced by Silverstripe CMS Ltd. In
```
[the remark this answers is not in the document]
Comment (Michael-HEIW) [reply docs/en/00_Getting_Started/02_Composer.md:326]: Silverstripe CMS Ltd changed to Silverstripe Ltd.

File: docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md (hunk 3 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/02_Controllers/06_Builtin_Middlewares.md:3]: Silverstripe CMS Framework > Silverstripe CMS

File: docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md (hunk 5 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md:17]: Framework and CMS removed

File: docs/en/02_Developer_Guides/14_Files/05_File_Migration.md (hunk 7 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/14_Files/05_File_Migration.md:93]: Changed Silverstripe CMS Platform to Silverstripe Cloud 👍

File: docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/index.md (hunk 8 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/index.md:8]: Removed Framework 👍

File: docs/en/02_Developer_Guides/02_Controllers/02_Routing.md (hunk 2 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/02_Controllers/02_Routing.md:285]: Removed Framework from Silverstripe CMS Framework (3 files)

File: docs/en/02_Developer_Guides/00_Model/12_Indexes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/00_Model/12_Indexes.md:20]: Removed Framework from Silverstripe CMS Framework (3 files)

File: docs/en/02_Developer_Guides/02_Controllers/02_Routing.md (hunk 2 above)
Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/02_Controllers/02_Routing.md:285]: Removed Framework from Silverstripe CMS Framework

Comment (Michael-HEIW) [reply docs/en/02_Developer_Guides/02_Controllers/02_Routing.md]: Silver**S**tripe CMS changed to Silverstripe CMS (Framework previously removed)

**Comment by NightJar** [comment]
Ideal, nice work @Michael-HEIW :D !

**Pull request merged**